### PR TITLE
PHP: update readme file

### PR DESCRIPTION
**Pull request by ZhouyihaiDing**
For the issue #12481, https://github.com/grpc/grpc.github.io/issues/579.

- Add links to grpc.io and cloud.google.com
- Make the installation part the same as [grpc.io](https://github.com/grpc/grpc.github.io/pull/593)
- Update the README file for php example.

### CODE REVIEW

**Comment by ZhouyihaiDing** [comment]
Update all related version in README file. I think this PR can be merged after the test finish.

**Comment by ZhouyihaiDing** [comment]
Bazel Debug build for C/C++- #15610
Bazel Opt build for C/C++ - #15610

**Comment by ZhouyihaiDing** [comment]
Hi @stanley-cheung . I updated all related version in README file. Thanks for reviewing it!

File: src/php/README.md [hunk 1]
```diff
@@ -124,44 +115,71 @@ $ make
...
+v1.3.4 | 3.3.0 
+v1.3.5 | 3.2.0
+v1.4.0 | 3.3.0 
+v1.6.0 | 3.4.0
+v1.8.0 | 3.5.0
+v1.12.0 | 3.5.2
 
 If `protoc` hasn't been installed, you can download the `protoc` binaries from
 [the protocol buffers Github repository](https://github.com/google/protobuf/releases).
+Then unzip this file and Update the environment variable `PATH` to include the path to
```
Comment (stanley-cheung) [inline src/php/README.md]: Update -> update

File: src/php/README.md [hunk 2]
```diff
@@ -124,44 +115,71 @@ $ make
...
+v1.4.0 | 3.3.0 
+v1.6.0 | 3.4.0
+v1.8.0 | 3.5.0
+v1.12.0 | 3.5.2
 
 If `protoc` hasn't been installed, you can download the `protoc` binaries from
 [the protocol buffers Github repository](https://github.com/google/protobuf/releases).
+Then unzip this file and Update the environment variable `PATH` to include the path to 
+the protoc binary file./protobuf/releases).
+Then unzip this file and Update the environment variable `PATH` to include the path to
```
Comment (stanley-cheung) [inline src/php/README.md]: Ditto

File: src/php/README.md [hunk 3]
```diff
@@ -173,32 +191,47 @@ $ ./autogen.sh && ./configure && make
...
 $ sudo pecl install protobuf
 ```
+or specific version
 
-2. PHP implementation (for easier installation)
+``` sh
+$ sudo pecl install protobuf-3.5.1.1
+```
 
+After protobuf extension is installed, Update php.ini by adding this line
```
Comment (stanley-cheung) [inline src/php/README.md]: Ditto

File: src/php/README.md (hunk 1 above)
Comment (ZhouyihaiDing) [reply src/php/README.md]: Done.

File: src/php/README.md (hunk 2 above)
Comment (ZhouyihaiDing) [reply src/php/README.md]: Done

File: src/php/README.md (hunk 3 above)
Comment (ZhouyihaiDing) [reply src/php/README.md]: Thanks! Done!

**Pull request merged**